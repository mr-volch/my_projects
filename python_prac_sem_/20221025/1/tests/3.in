print(tuple(fib(90,195))[5:10])
