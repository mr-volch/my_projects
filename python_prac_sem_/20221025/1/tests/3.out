(51680708854858323072, 83621143489848422977, 135301852344706746049, 218922995834555169026, 354224848179261915075)
